pkg/granted/credentialsprocess.go

```diff
@@ -30,12 +31,14 @@ var CredentialsProcess = cli.Command{
 	Usage:       "",
 	Hidden:      true,
 	Subcommands: []*cli.Command{&ConfigSetup},
-	Flags:       []cli.Flag{&cli.StringFlag{Name: "profile"}, &cli.StringFlag{Name: "rule"}},
+	Flags:       []cli.Flag{&cli.StringFlag{Name: "profile", Required: true}, &cli.StringFlag{Name: "rule"}},
 	Action: func(c *cli.Context) error {
 
 		url := ""
 		profileName := c.String("profile")
-		// ruleId := c.String("rule")
+		if !strings.HasPrefix(profileName, "granted.") {
+			log.Fatal("Profile name must start with 'granted.' prefix. Check your config file to verify.")
+		}
```

Comment:
can we remove this check? if this function is being called, it means that the user has correctly set up their credential process, so it doesn't matter what their profile is named. This forces advanced users who might want to name their profiles a different way to use our `granted.` naming scheme.

```suggestion
```